packages/ui/src/components/va-data-table/VaDataTable.stories.ts

```diff
@@ -1,12 +1,45 @@
+import { defineVaDataTableColumns, defineVaDataTableItems } from './fabrics'
 import { defineComponent } from 'vue'
-import VaDataTable from './VaDataTable.demo.vue'
+import VaDataTableDemo from './VaDataTable.demo.vue'
+import { VaDataTable } from '..'
 
 export default {
   title: 'VaDataTable',
-  component: VaDataTable,
+  component: VaDataTableDemo,
 }
 
+const columns = defineVaDataTableColumns([
+  { key: 'name' },
+  { key: 'email' },
+])
+
+const items = defineVaDataTableItems<typeof columns>([
```

Comment:
Create satisfies VaDataTableColumns<Items> and satisfies VaDataTableItems<Columns>